src/aat/resources/features/F-104/S-635.td.json

```diff
@@ -142,7 +143,8 @@
             "[SECURITY_CLASSIFICATION]": "PRIVATE",
             "[CASE_TYPE]": "AAT_PRIVATE",
             "[LAST_MODIFIED_DATE]": "[[ANYTHING_PRESENT]]"
-          }
+          },
+          "supplementary_data" : null
```

Comment:
`"supplementary_data" : null`  is repeated quite often in the various test files. Is it possible to move it up one level so it's inherited everywhere? I think @Abi-Laver worked on a similar enhancement she would know